tests/test_sparkkernelbase.py

```diff
@@ -284,7 +282,6 @@ def test_execute_throws_if_fatal_error_happened():
         # Assertions
         assert kernel._fatal_error == fatal_error
         assert execute_cell_mock.call_count == 0
-        assert send_error_mock.call_count == 1
```

Comment:
This assertion is still needed. It's just that you would now verify with the ipython_display mock.
You might need to add `ipython_display=None` to SparkKernelBase constructor.